<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdPolarTransformDialog</class>
 <widget class="QDialog" name="QxrdPolarTransformDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polar Transform Current Image...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="6" column="2">
    <widget class="QSpinBox" name="m_Oversample">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Polar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Radial</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="m_RadialType">
       <item>
        <property name="text">
         <string>Integrate vs TTH</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Integrate vs Q</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Integrate vs R</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="m_PolarStep">
       <property name="specialValueText">
        <string>Fixed # Steps</string>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="m_RadialStep">
       <property name="specialValueText">
        <string>Fixed # Steps</string>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string># Steps</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="m_PolarStart">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>720.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="m_RadialStart">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="m_PolarEnd">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>720.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="m_RadialEnd">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="m_PolarNSteps">
       <property name="specialValueText">
        <string>Fixed Size</string>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="m_RadialNSteps">
       <property name="specialValueText">
        <string>Fixed Size</string>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="m_PolarType">
       <item>
        <property name="text">
         <string>No Integration</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Integrate vs Chi</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Output Type:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="m_EnableGeometry">
     <property name="toolTip">
      <string>Enable geometry corrections during integration
intensity *= pow(distance/m_DetectorDistance, 2)/cos(tth);</string>
     </property>
     <property name="text">
      <string>Enable Geometry Corrections </string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Oversample Factor</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="m_OutputType">
     <item>
      <property name="text">
       <string>No Output</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Data in 'Destination'</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Radial Plot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Polar Plot</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>268</width>
       <height>119</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="m_Destination"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Destination:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="m_EnablePolarization">
     <property name="toolTip">
      <string>Enable polarization correction during integration:
intensity *= (m_Polarization      *(1 - pow(sin(chi)*sin(tth),2)) +
                   (1 - m_Polarization)*(1 - pow(cos(chi)*sin(tth),2)));</string>
     </property>
     <property name="text">
      <string>Enable Polarization Corrections </string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="m_Polarization">
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Polarization Factor</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QxrdPolarTransformDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QxrdPolarTransformDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
